### DISCUSSION TITLE
Windows Thumbnails [Taskbar effect]

### DISCUSSION THREAD

**Discussion by maplee0** [lxqt/lxqt#2284] [Ideas]
Hello! I've used kde for a decade, but need something lightweight for a laptop. There is one very important effect in kde4+ it's windows thumbnails above tasks. Really, it's very comfortable to use. I think the old taskbar style (win95-xp-vista) is not good as the modern one (win7+) and that was in kde4 and now it's included in kde5. Unfortunately, when kde5 was actively developed, it was moved to plasma. Yes, compiz has it, but that WM is very buggy and bloat. Recently, I was surprised in a good way, because tint2 has it since 2018. It works, picom should be enabled to get it, and it's disabled in the default settings. And that is a gtk2 panel, not even a DE. I suppose LXQt would be more popular if it was optionally implemented and more people would use your nice DE. I already noticed a similar suggestion a few years ago, but that one was created in 2014, and then archived. Thank you all for the great DE, have always liked LXQt!

**Comment by tsujan** [comment]
> I suppose LXQt would be more popular if it was optionally implemented

It can't be implemented in LXQt because, as you seem to know already, such thumbnails are created by WM. LXQt is WM-agnostic. If a WM creates such thumbnails in a DE-independent way (like Compiz does), they'll appear in LXQt too.

**Reply by stefonarch** [reply]
How does tint2 do it then if it's only a panel?

**Reply by tsujan** [reply]
> How does tint2 do it then if it's only a panel?

I don't know about tint2, but It can be done by X11 codes, like some WMs do it. But we don't want that in LXQt.

**Reply by stefonarch** [reply]
I see, no X11 code for sure. 

Personally I never understood the sense of those, I tried them but I know how my thunderbird or pcmanfm-qt  windows look -  thumbnails are useful for some 2315.jpg file. But I guess its the same with icons versus text: I couldn't use also the pc without text besides or under icons.

**Reply by tsujan** [reply]
It their creation had no cost, why not? But, apart from avoiding X11 code, we don't want to bloat LXQt; most users count on that.

**Reply by stefonarch** [reply]
No I didn't mean their useless only because there are it for me.

**Comment by maplee0** [comment]
Thank you for the answers! I appreciate your opinions.
